Documentation for submitting jobs to IonQ and Quantinuum (#431)
NVIDIA/cuda-quantum: python/README.md

# Welcome to the CUDA Quantum Python API

## Installing CUDA Quantum

> To pull down the CUDA Quantum Python wheel distribution, simply run:

Reviewer: ```suggestion
CUDA Quantum Python wheels are available on [PyPI.org](https://pypi.org/project/cuda-quantum). To install the latest release, simply run
```

# Welcome to the CUDA Quantum Python API

## Installing CUDA Quantum

To pull down the CUDA Quantum Python wheel distribution, simply run:

```console
pip install cuda-quantum
```

At this time, wheels are distributed for Linux operating systems only.

To build a source distribution of the Python API through pip:

```console
git clone https://github.com/NVIDIA/cuda-quantum.git
cd cuda-quantum

> pip install . 

Reviewer: ```suggestion
./scripts/install_prerequisites.sh && pip install . 
```

```

If you would like to build the entire C++ and Python API's from source,
follow the [installation instructions][official_install]. You
may then export your python path to the installation directory:

```console
export PYTHONPATH=$PYTHONPATH:/path/to/cudaq/install
```

[official_install]: https://nvidia.github.io/cuda-quantum/latest/install.html

## Running CUDA Quantum

You should now be able to import CUDA Quantum and start building quantum programs in
Python!

```console
import cudaq

kernel = cudaq.make_kernel()
qubit = kernel.qalloc()
kernel.x(qubit)
kernel.mz()

result = cudaq.sample(kernel)
```

## Documentation 

To see more examples, go to [python examples][python_examples], or check out the
[Python API reference][python_api_reference].

[python_examples]: https://nvidia.github.io/cuda-quantum/latest/using/python.html
[python_api_reference]: https://nvidia.github.io/cuda-quantum/latest/api/languages/python_api.html

## Contributing

There are many ways in which you can get involved with CUDA Quantum. If you are

> interested in developing quantum applications with CUDA Quantum, this repository

Reviewer: all the "this repository" in the readme needs to be replaced. The license section in particular doesn't seem applicable. Remove the section about the CLA bot. You can state the source code is available on GitHub and licensed under Apache 2.0. Let's maybe mirror the phrasing on NGC and also state 

"CUDA Quantum is an open source project. The source code is available [on GitHub](https://github.com/NVIDIA/cuda-quantum) and licensed under [Apache 2.0](https://github.com/NVIDIA/cuda-quantum/blob/main/LICENSE) license. By downloading NVIDIA CUDA Quantum, you accept the terms and conditions of the NVIDIA End User License Agreement. "

Author: Ok I've updated everything but you should definitely give them another look over when you can

import cudaq

kernel = cudaq.make_kernel()
qubit = kernel.qalloc()
kernel.x(qubit)
kernel.mz()

result = cudaq.sample(kernel)
```

## Documentation

To see more examples, go to [python examples][python_examples], or check out the
[Python API reference][python_api_reference].

[python_examples]: https://nvidia.github.io/cuda-quantum/latest/using/python.html
[python_api_reference]: https://nvidia.github.io/cuda-quantum/latest/api/languages/python_api.html

## Contributing

There are many ways in which you can get involved with CUDA Quantum. If you are
interested in developing quantum applications with CUDA Quantum, our [github][github_link]
is a great place to get started! For more information about contributing to the
CUDA Quantum platform, please take a look at
[Contributing.md](../Contributing.md).

## License

> CUDA Quantum is an open source project. The source code is avilable on 

Reviewer: ```suggestion
CUDA Quantum is an open source project. The source code is avilable on
```

[github][github_link] and licensed under [Apache License 2.0](../LICENSE).
By downloading NVIDIA CUDA Quantum, you accept the terms and conditions of
the NVIDIA End User License Agreement.

[github_link]: https://github.com/NVIDIA/cuda-quantum/

## Feedback

Please let us know your feedback and ideas for the CUDA Quantum platform in the

> [Discussions][cuda_quantum_discussions] tab of our [github][github], or file an

Reviewer: ```suggestion
[Discussions][discussions] tab of our [GitHub repository][github], or file an
```

> [issue][cuda_quantum_issues]. To report security concerns or [Code of

Reviewer: ```suggestion
[issue][cuda_quantum_issues]. To report security concerns please reach out to
```

> Conduct](../Code_of_Conduct.md) violations, please reach out to

Reviewer: ```suggestion
```

> [cuda-quantum@nvidia.com](mailto:cuda-quantum@nvidia.com).

Reviewer: ```suggestion
out to [cuda-quantum@nvidia.com](mailto:cuda-quantum@nvidia.com).
```

> [cuda_quantum_discussions]: https://github.com/NVIDIA/cuda-quantum/discussions

Reviewer: ```suggestion
[discussions]: https://github.com/NVIDIA/cuda-quantum/discussions
```

> [cuda_quantum_issues]: https://github.com/NVIDIA/cuda-quantum/issues

Reviewer: ```suggestion
[cuda_quantum_issues]: https://github.com/NVIDIA/cuda-quantum/issues
[github_repo]: https://github.com/NVIDIA/cuda-quantum
```
